aries-site/src/pages/components/checkbox.js

```diff
@@ -19,16 +27,143 @@ const CheckBox = () => (
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
         <SubsectionText>
-          When the user needs to select one or more options, use a checkbox. The
-          click target should include the checkbox label.
+          {page.description}
+        </SubsectionText>
+        <Example
+          docs="https://v2.grommet.io/checkbox?theme=hpe#props"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/checkbox/CheckboxSimpleExample.js"
+          figma="https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1745%3A43"
+          designer="https://designer.grommet.io/home?id=HPE-design-system-taylor-seamans-hpe-com"
+        >
+          <CheckBoxSimpleExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBox can be used to present an individual option or group of
+          options to the user. When presented in a group, the user can select a
+          single or multiple options. Refer to CheckBoxGroup for examples.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Usage" level={3} gap="small">
+        <SubsectionText>
+          CheckBox can be used in the form of a{' '}
+          <Anchor href="#toggle">toggle</Anchor>, a single option which can
+          indicate something such as on or off. Toggle is prefered to quickly
+          switch between two possible states. Toggles usually provides immediate
+          results in which gives users the ability to control their preference.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox should not be used if a user should only be allowed to select
+          one option from a list of options. In the case that a user should only
+          be able to select one option, refer to{' '}
+          <Anchor href={nameToPath('RadioButton')}>RadioButton</Anchor>.
+        </SubsectionText>
+        <SubsectionText>
+          There are three states which CheckBox provides:
+          <BulletedList
+            level={3}
+            items={[
+              'Unchecked - Value has not been selected.',
+              `Checked - Value has been selected. Also, the checked state may
```

Comment:
Not sure we need the second sentence in the Checked state -- feels a little confusing and seems like that content is handled in Indeterminate.